dotCMS/src/main/java/com/dotmarketing/portlets/containers/model/ContainerView.java

```diff
@@ -0,0 +1,49 @@
+package com.dotmarketing.portlets.containers.model;
+
+import com.dotmarketing.beans.Host;
+
+import com.dotmarketing.portlets.containers.business.FileAssetContainerUtil;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+
+/**
+ * It is a {@link Container} for UI, it make sure to return the container relative path from {@link ContainerView#host}
+ */
+@JsonSerialize(using = ContainerViewSerializer.class)
+public class ContainerView {
+    private Container container;
+    private Host host;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'host' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)